Objects/weakrefobject.c

```diff
@@ -128,19 +130,19 @@ gc_clear(PyWeakReference *self)
 
 
 static PyObject *
-weakref_call(PyWeakReference *self, PyObject *args, PyObject *kw)
+weakref_vectorcall(PyWeakReference *self, PyObject *const *args,
+                   size_t nargsf, PyObject *kwnames)
 {
-    static char *kwlist[] = {NULL};
-
-    if (PyArg_ParseTupleAndKeywords(args, kw, ":__call__", kwlist)) {
-        PyObject *object = PyWeakref_GET_OBJECT(self);
-        Py_INCREF(object);
-        return (object);
+    if (!_PyArg_NoKwnames("__call__", kwnames)) {
```

Comment:
Mmmm. the first parameter should be the function name. Should it be "weakref"? Same remark for _PyArg_CheckPositional().